azure/cargo-clippy.yml

```diff
@@ -2,13 +2,20 @@ parameters:
   rust: stable
   allow_fail: false
   setup: []
+  fail_all: false
```

Comment:
What is the intent behind `fail_all`?  It seems to add warnings but the parameter name doesn't imply that to me.

Reply:
`-D warnings` turns all warnings into hard errors. `fail_all` was intended to indicate that "this will fail on any and all issues", but maybe there's a better wording?

Reply:
`include_warnings`?

Reply:
Hmm.. It's not that it _includes_ warnings, but rather that it treats all warnings as errors. `warnings_as_errors` is pretty verbose though, so I'd prefer to find something shorter.

Reply:
`include_warnings` as in "check clippy and warnings" though I also see how it doesn't clarify they are errors.  Unsure if its needed?